### PR TITLE
Create ui-c8y-1020-0-0-use-routing-in-alarms-6264.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by janh

Ticket: [MTM-59106](https://cumulocity.atlassian.net/browse/MTM-59106)
Original PR: [6264](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6264)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-use-routing-in-alarms-6264.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Use routing in alarms (#6264)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-use-routing-in-alarms-6264.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Alarm notifications can be routed to specific users based on alarm type
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-use-routing-in-alarms-6264.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59106
+version: 1020.0.0
+---
+Use routing in alarms (#6264)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-use-routing-in-alarms-6264.md:17]: The Claude AI suggested to use the following body:
```suggestion
In large IoT deployments with many different types of devices and assets, it is important that the right people are notified about critical issues. With this update, alarm notifications can now be routed to specific users or user groups based on the alarm type. For example, all battery related alarms can be sent to the team responsible for device maintenance. This improves the alarm notification process and ensures critical issues are resolved faster. To take advantage of this new capability, the alarm configuration needs to be updated to specify the routing for each alarm type.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by janhommes — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-use-routing-in-alarms-6264.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59106
+version: 1020.0.0
+---
+Alarms previously could not be shared, as they had no unique link to reference them e.g. in mails or chats. With this change, each alarm gets an unique link, helping to sharing and navigating between alarms more easily.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-use-routing-in-alarms-6264.md:17]:
```suggestion
Previously, alarms could not be shared as they had no unique link to reference them, for example in mails or chats. With this change, each alarm gets an unique link. This allows to share alarms and navigate between alarms more easily.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rahultsag As this is a new feature, please review and approve the PR.

**Review by rahultsag — APPROVED** [review]

**Pull request merged**